.github/workflows/scripts/update_components.sh

```diff
@@ -76,7 +78,11 @@ function get_version() {
     if [[ "${push}" == true ]]; then
         echo "push"
         git push --set-upstream origin "${PR_BRANCH}"
+        numberOfChanges+=$((numberOfChanges + 1))
```

Comment:
```suggestion
        numberOfChanges+=1
```
This would be sufficient, if the idea is just to increment numberOfChanges var